docs/policy-management.md

```diff
@@ -0,0 +1,213 @@
+2. Parse and validate the body, with the same procedure used in the POST endpoint. First, we perform the basic sanitization checks. Upon success, extra checks are performed to see if the new definition stays within the scope of the client:
+     - Check that the newly defined policy does not define other policies
+     - Check that the new policy does not contain any rules that do not belong to the client
+     - Check that no unrelated quads to the policy and its rules are added. 
+     
+    Failed checks will result in a response with **status code 400** and a dedicated message.
+3. Delete the old policy, but keep a copy for a possible rollback. The deletion uses the procedure used in the [DELETE](#deleting-policies) endpoint.
+4. Add the new policy. On success, the server will respond with **status code 200** and a body containing the new policy and its rules (within scope of the client). When this does not succeed, a rollback will be set up:
+    - The server will try to reset the state of the policy by adding the old quads. If this succeeds, an internal server error with **status code 500** will indicate that nothing has been rewritten, and the old version is restored.
+    - When the rollback fails, we basically deleted the policy information within our reach. An internal server error with **status code 500** will indicate this.
+
+Note that this endpoint uses the POST and DELETE functionality to implement the PUT.
+
+#### PATCH
+A PATCH request will update the specified policy. The request expects a body with content type [`application/sparql-query`](https://www.w3.org/TR/rdf-sparql-query/). The INSERT and DELETE properties can be used to modify the requested policy. These modifications can only be applied to parts of the policy within the client's scope.
+
+The PATCH process:
+1. Collect all information about the existing policy with the procedure explained in the [GET One Policy endpoint](#get-one-policy). If the policy does not exist, the server responds with **status code 400**. In the current implementation, we have chosen that the client cannot PATCH a policy in which it has no rules. Doing so will also result in a response with **status code 400**.
+2. The content type of the body gets validated. The content-type must be set to `application/sparql-query`. Any other type will result in a response with **status code 400**.
+3. We use the policy information to create a store which only contains groups **(1)**, **(3)** and **(4)** as explained [above](#get-one-policy). This will serve as an isolated store, on which we can execute the update query. This implementation has its advantages:
+    - We do not need to validate the sparql query, since we execute it on an isolated store.
+    - Performing DELETE queries on rules out of your scope will simply not work, since they are not part of the isolated store.
```

Comment:
Does this mean that nothing then will happen?
I understand it as such and think that is valid behaviour.